fe-nc-news-react/src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { getAllArticles } from "../../api";
+import Article from "./Article";
+
+function ArticleList(props) {
+  const { articles, setArticles } = props;
+  const [isError, setisError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getAllArticles()
+      .then((allArticles) => {
+        setArticles(allArticles);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+        setisError(true);
+      });
+  });
```

Comment:
You are missing  dependancy array. This could risk an infinite loop! Please add one